modules/data-entry/src/main/frontend/src/vocabQuery/queryStyle.jsx

```diff
@@ -109,6 +109,15 @@ const thesaurusStyle = theme => ({
   infoDialog: {
     zIndex: "1350 !important",
   },
+  noResults: {
+    margin: theme.spacing(-1,0)
+  },
+  noneOfAboveResultsText: {
+    marginBottom: theme.spacing(-1),
+  },
+  noneOfAboveResults: {
+    borderTop: "2px solid " + grayColor[5]
+  }
```

Comment:
Replace this styling with a `<Divider />` inserted between the list of suggestions and the "None of the above" option.

```suggestion
  noneOfAboveResults: {
    borderTop: "2px solid " + grayColor[5]
  }
```
Also, with the introduction of the `Divider`, the two classes above can now be merged to simplify styling.

Reply:
Done in 882eeec